healthcare/v1beta1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceConditionalUpdate.java

```diff
@@ -17,57 +17,80 @@
 package snippets.healthcare.fhir.resources;
 
 // [START healthcare_conditional_update_resource]
+
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.http.HttpHeaders;
 import com.google.api.client.http.HttpRequestInitializer;
 import com.google.api.client.http.javanet.NetHttpTransport;
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.healthcare.v1beta1.CloudHealthcare;
-import com.google.api.services.healthcare.v1beta1.CloudHealthcare.Projects.Locations.Datasets.FhirStores.Fhir;
 import com.google.api.services.healthcare.v1beta1.CloudHealthcareScopes;
 import com.google.api.services.healthcare.v1beta1.model.HttpBody;
+
 import java.io.IOException;
+import java.net.URISyntaxException;
 import java.util.Collections;
 
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+
 public class FhirResourceConditionalUpdate {
   private static final String FHIR_NAME =
       "projects/%s/locations/%s/datasets/%s/fhirStores/%s/fhir/%s";
   private static final JsonFactory JSON_FACTORY = new JacksonFactory();
   private static final NetHttpTransport HTTP_TRANSPORT = new NetHttpTransport();
 
-  public static void fhirResourceConditionalUpdate(String resourceName) throws IOException {
+  public static void fhirResourceConditionalUpdate(
+      String fhirStoreName,
+      String resourceType,
+      String data) throws IOException, URISyntaxException {
     // String resourceName =
     //    String.format(
     //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "fhir-id");
-    String resourceType = "Patient";
+    // String resourceType = "Patient";
+    // String data = "{'family': 'Smith'}";
 
     // Initialize the client, which will be used to interact with the service.
     CloudHealthcare client = createClient();
 
-    // Fetch the initial state of the resource.
-    Fhir.Get getRequest =
-        client.projects().locations().datasets().fhirStores().fhir().get(resourceName);
-    HttpBody resource = getRequest.execute();
+    HttpClient httpClient = HttpClients.createDefault();
```

Comment:
I don't know if there is a bug tracking this issue, but maybe it should be linked to in the comment? So future readers understand why these samples aren't using the client?

Reply:
There isn't currently a bug tracking it but I can create one. I doubt they're going to prioritize it though.

Reply:
Probably a good idea, just to make sure it's somewhere on their radar.

Reply:
Created bug 133419634